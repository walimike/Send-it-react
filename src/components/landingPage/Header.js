import React, { Component } from "react";

class Header extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
        <div className='head-wrapper'>
          <a href="#" className="app-logo">
          <img src="https://image.shutterstock.com/image-vector/running-man-postal-box-logomessenger-260nw-293379359.jpg" />
            <span>Send It</span>
          </a>
          <ul id="nav-mobile" className="right hide-on-med-and-down_">
            <li className='action-buttons'>
            <a className="waves-effect waves-light btn-small join"
                onClick={this.props.openSignUp} >Join</a>
            <a className="waves-effect waves-light btn-small login"
            onClick = {this.props.openLogIn}>login</a> </li>
          </ul>
        </div>
    );
  }
}

export default Header;
