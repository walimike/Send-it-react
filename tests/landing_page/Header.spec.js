import React from "react";
import { shallow } from "enzyme";

import Header from "../../src/components/landingPage/Header";

describe("Header", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
