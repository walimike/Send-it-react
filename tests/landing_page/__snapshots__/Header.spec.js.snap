// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without crashing 1`] = `ShallowWrapper {}`;
